app/lib/controllers/chat_controller.dart

```diff
@@ -303,7 +303,10 @@ class ChatController extends GetxController {
       FileDescription? description = message.fileDescription();
       if (description != null) {
         types.FileMessage m = types.FileMessage(
-          author: types.User(id: message.sender()),
+          author: types.User(
+            id: message.sender(),
+            firstName: getNameFromId(message.sender()),
+          ),
```

Comment:
I think we should do the same for pictures. Because when sending a picture, we don't see the avatar.